src/crimson/os/seastore/async_cleaner.h

```diff
@@ -1207,6 +1194,9 @@ class AsyncCleaner : public SegmentProvider, public JournalTrimmer {
    * Segments calculations
    */
   std::size_t get_segments_in_journal() const {
+    if (journal_type == journal_type_t::CIRCULAR) {
+      return 0;
+    }
```

Comment:
@myoungwon The root issue is that there are no segments in journal with CircularBoundedJournal.

Reply:
Yeah, I originally added this line before, but your comment is that I need to put this kind of condition to more general place. Anyway, this is ok for now.

Reply:
Sorry, I'm not aware of the exact issue until I tested it myself.

Reply:
No worries. Anyway, I need to time to look at this if my review is needed.

Reply:
Reviews are always valuable!

Feel free to also comment wherever the intention is not clear, so I can elaborate.